python/tests/helpers/io/TemporaryAaCTestFile.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
```

Comment:
This test helper class provides an easy way to set the suffix of test files to '.aac' in order to account for the new behavior that ignores files that don't end in '.yaml' or '.aac'